packages/instrumentation-node-cache/src/types.ts

```diff
@@ -0,0 +1,14 @@
+import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+import { Span } from '@opentelemetry/api';
+
+export interface NodeCacheRequestInfo {
+    moduleVersion: string;
+    operation: string;
+    args: any[];
+}
+
+export interface NodeCacheInstrumentationConfig extends InstrumentationConfig {
+    requestHook?: (span: Span, requestInfo: NodeCacheRequestInfo) => void;
+    responseHook?: (span: Span, response: any) => void;
```

Comment:
I think that if we use `requestInfo` we should also use `responseInfo`, even if it currently only contains the response object

Reply:
fixed